content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -59,13 +60,21 @@ to consume and react to them. A live list of plugins and their descriptions is h
 website](https://prow.ci.openshift.org/plugins), please consult that list while reading the following section for more
 detail.
 
-Plugin configuration is stored in
+In order to create a new Plugin configuration for your repository the following target can be used:
```

Comment:
Only if the main file has config for that repo

Reply:
@alvaroaleman can you help me understand this a little better? Are you saying that the `make prow-config` target is only useful if there is already a configuration for the repo in the `_plugins.yaml` file? or vice versa?

Reply:
`make prow-config` will shard the main plugin config file but if that does not contain any config for your repo, the file will not get generated. Maybe somehow make it clear that `make prow-config` will only generate a file if run after `make new-repo`?

Reply:
Ah that makes sense, I wasn't completely aware of everything that was doing. I will try to call that out.